Makefile

```diff
@@ -173,7 +175,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 .PHONY: docker-build
-docker-build: check ## Build docker image with the manager.
+docker-build: test
```

Comment:
Why do you want to remove the `check` target?

Reply:
Locally it's simpler to build without `test`, but this is a valid point so I've changed it so CI build will still use `check` while adding another option for a build with `test`

Reply:
WFM